File: FRONTEND/src/models/Core/FunctionInfo.js
import BaseModel from '../BaseModel';
import ModuleInfo from './ModuleInfo';

export default class FunctionInfo extends BaseModel {
  static entity = 'functioninfo'
  static entityName = "functioninfo";
  
  static fields () {
    return {
      id : this.uid(),
      name : this.string(null).nullable(),
      uri : this.string(null).nullable(),
      iconName : this.string(null).nullable(),
      isEnabled : this.boolean(null).nullable(),
      moduleInfoId : this.string(null).nullable(),
      moduleInfo : this.belongsTo(ModuleInfo, "moduleInfoId"),
    }
  }
}

File: BACKEND/Tutorial/src/ApplicationCore/Entities/Framework/FunctionInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Tutorial.ApplicationCore.Entities
{
	public class FunctionInfo : BaseEntity
	{
		public string Id { get; set; }
		public string Name { get; set; }
		public string Uri { get; set; }
		public string IconName { get; set; }
		public bool IsEnabled { get; set; }
		public int? ModuleInfoId { get; set; }
		public virtual ModuleInfo ModuleInfo { get; set; }
	}
}
